src/pages/rekey-reason/rekey-reason.ts

```diff
@@ -122,4 +203,47 @@ export class RekeyReasonPage extends PracticeableBasePageComponent {
     }
   }
 
+  ipadIssueSelected() {
+    console.log('asdasdasdASSAASD');
+    this.store$.dispatch(new IpadIssueSelected(this.formGroup.controls['ipadIssue'].value));
+  }
+  ipadIssueTechFaultChanged() {
+    this.store$.dispatch(new IpadIssueTechFault(this.formGroup.controls['ipadIssueTechFault'].value === 'true'));
```

Comment:
Could you place the value of the control `ipadIssueTechFault` in the arguments of this function so that you don't need to check every time.

Just send it back from the html template.